packages/gatsby-theme-newrelic/gatsby-browser.js

```diff
@@ -17,5 +18,25 @@ export const wrapRootElement = ({ element }, pluginOptions) => {
   );
 };
 
+export const onClientEntry = (_, { newrelic, i18n }) => {
+  if (newrelic && window.newrelic) {
```

Comment:
Out of curiosity, is this `newrelic` the theme's config value? If so, I'd be curious why we might depend on that to add custom attributes. Would `window.newrelic` be enough?

Reply:
Very true! `window.newrelic` should be enough.